packages/shared/akamai-edgeworker-sdk/src/featureStore/index.ts

```diff
@@ -59,8 +59,14 @@ export class EdgeFeatureStore implements LDFeatureStore {
         case 'segments':
           callback(item.segments[dataKey]);
           break;
+        case 'configurationOverrides':
+          callback(item.configurationOverrides?.[dataKey] ?? null);
+          break;
+        case 'metrics':
+          callback(item.metrics?.[dataKey] ?? null);
```

Comment:
Is there a difference between passing `null` and `undefined`? Usually there's no difference:

```suggestion
          callback(item.configurationOverrides?.[dataKey]);
          break;
        case 'metrics':
          callback(item.metrics?.[dataKey]);
```

Reply:
So, functionally not a lot in this specific case. Here though the LDFeatureStore is still compatible with the one we had before, and the typing had `LDFeatureStoreItem | null`. So it just has 'null' here to keep typescript  compiling. Without it it simply would not compile.

At some point we may change that typing. I have to release 9.0 for migrations, so it would possibly be a good time. 

Not in this situation, but in regards to truthiness, it can be very important. Thankfully `?.` and `??` have reduced it.

If 0/"" is valid, but null/undefined are not, then you often would end up with checks that check for both.